### ISSUE TITLE
Neighborhood Name Displayed is Not Current Neighborhood

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#308]
Just received this email:

> When I first logged in today, I was placed here. As you can see, it says I'm in Twining, but that's definitely the National Gallery... I also couldn't finish this segment.

![image](https://cloud.githubusercontent.com/assets/1621749/18258059/a5e9eaa6-739c-11e6-8e99-e163e7327565.png)

This user also left a comment using the feedback tool so we have the panorama stored and when this happened--maybe useful for tracking down what happened:

![image](https://cloud.githubusercontent.com/assets/1621749/18258070/ce929502-739c-11e6-8acb-caae27c278c6.png)

**Comment by kotarohara** [comment]
I originally thought this is because of the underlying data. For example, a census tract polygon that we use to represent a neighborhood contains multiple candidates for neighborhood names (which comes from another DC DOT data source) and we end up taking the one that does not match what people consider as the right neighborhood name. But I checked the location of Twining and that's pretty far away from the DC Downtown where the National Gallery is situated, so now I think this is not the issue.

I think the issue is that the neighborhood name is not getting updated after the user have moved to a new neighborhood (e.g., on completing audits for all the street segments in a neighborhood). If that's the case, it should have been fixed in the PR #312 .
